designer/server/src/main/scala/pl/touk/nussknacker/ui/component/ComponentService.scala

```diff
@@ -65,29 +66,12 @@ class DefaultComponentService(
 
   import cats.syntax.traverse._
 
-  private def processingTypeAndComponentInfoToComponentId(
+  private def processingTypeToNonFragmentComponents(
```

Comment:
1. Please write a comment that we collect all components but fragments because fragments can't have ComponentId overridden, so we can use the default one without fetching them. 
2. Please move this method below the last invocation of it
3. After 2., please also move  `definedComponents` below it's last invocation.
Thanks to 2. and 3., we reduce the number of jumps thought the file during further code analysis